source/agora/network/rest_api.d

```diff
@@ -80,8 +118,12 @@ public struct NetworkInfo
 
 *******************************************************************************/
 @path("/")
-public interface API
+interface VibeRestAPI
 {
+    void metaAddPeer (string peer);
+    public void start ();
+    public void shutdown ();
```

Comment:
Those don't belong to the public interface. Since this is the `VibeRestAPI`, you can't just expose `start` and `shutdown`...

Reply:
it is temporary so that the compilation does not break